### PR TITLE
fixed- error x

### PR DESCRIPTION
**Pull request by DiegoRosado9615**
Esto es una prueba

### CODE REVIEW

File: src/main/java/com/test/demo/SpringtestApplication.java [hunk 1]
```diff
@@ -10,7 +10,8 @@ public static void main(String[] args) {
 		SpringApplication.run(SpringtestApplication.class, args);
 		System.out.println("INICIANDO SPRING APP DIEGO");
 		System.out.println("Primer merge");
-		
+		System.out.println("XD");
+		System.out.println("Esto no va :C");
```
Comment (PredadorAkrid) [inline src/main/java/com/test/demo/SpringtestApplication.java:14]: Remove system.out.println

File: src/main/java/com/test/demo/SpringtestApplication.java [hunk 2]
```diff
@@ -10,7 +10,8 @@ public static void main(String[] args) {
 		SpringApplication.run(SpringtestApplication.class, args);
 		System.out.println("INICIANDO SPRING APP DIEGO");
 		System.out.println("Primer merge");
-		
+		System.out.println("XD");
```
Comment (PredadorAkrid) [inline src/main/java/com/test/demo/SpringtestApplication.java:13]: dasdasdasd

**Review by PredadorAkrid — COMMENTED** [review]
aaa

File: src/main/java/com/test/demo/SpringtestApplication.java (hunk 1 above)
Comment (DiegoRosado9615) [reply src/main/java/com/test/demo/SpringtestApplication.java]: Fixed at 110ea2a5edaf4c9e67f48752b9eb0d633f492523

File: src/main/java/com/test/demo/SpringtestApplication.java (hunk 2 above)
Comment (DiegoRosado9615) [reply src/main/java/com/test/demo/SpringtestApplication.java]: fixed at 110ea2a5edaf4c9e67f48752b9eb0d633f492523

**Pull request merged**